src/time-map.scm

```diff
@@ -0,0 +1,20 @@
+(use-modules ((rnrs) :version(6)))
```

Comment:
Hi Linas,
 I will fix that. I know i need to refactor the code and split files. I
will do those things too.
Regards,
Mandeep

On Sun, Jul 10, 2016 at 7:03 AM, Linas Vepštas <notifications@github.com>
wrote:

> In src/time-map.scm
> <https://github.com/opencog/ros-behavior-scripting/pull/56#discussion_r70173321>
> :
>
> > @@ -0,0 +1,20 @@
> > +(use-modules ((rnrs) :version(6)))
>
> please do not use rnrs if at all possible. There are much better ways to
> do things than to use that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencog/ros-behavior-scripting/pull/56/files/a08408aee9c48f6316bafd2a8c987ee4aa5b1030#r70173321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHM_f_ar_CmTZ3_UpnqKkaeUuVfJOMIkks5qUCjOgaJpZM4JIt6x>
> .
>